plugins/modules/ndo_port_channel_interface.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_port_channel_interface
+short_description: Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.2 (NDO v4.4) and later.
+author:
+- Gaspard Micol (@gmicol)
+options:
```

Comment:
All the attributes under options are over-indented. Can you reduce them by a tab?

Reply:
changed